src/main/java/appeng/client/ClientHelper.java

```diff
@@ -74,6 +77,9 @@
 
 public class ClientHelper extends ServerHelper
 {
+	private final static String KEY_CATEGORY = "key.appliedenergistics2.category";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)